tests/acceptance/src/test/java/org/activityinfo/test/sut/DevServerAccounts.java

```diff
@@ -69,9 +69,8 @@ public UserAccount ensureAccountExists(String email) {
     }
 
     private String connectionUrl() {
-        return String.format("jdbc:mysql://localhost/activityinfo_at?useUnicode=true&characterEncoding=UTF-8", 
```

Comment:
I think this broke the sonar build:
https://jenkins.bedatadriven.com/job/ActivityInfo/job/Server/job/Sonar/49/testReport/(root)/Scenario_%20Defining%20database-level%20target/Given_I_have_created_a_database__RRMP_/

If this source change was intended, can you take a look at the Jekins configuration?